src/Model/Profile.php

```diff
@@ -219,6 +222,12 @@ public static function load(App $a, string $nickname, bool $show_contacts = true
 			return [];
 		}
 
+		// System user, aborting
```

Comment:
Sadly for the HCard we need to load the profile ...

Reply:
`Profile::load` makes no sense for a technical endpoint since most of it is about setting the theme and updating the sidebar. A simple `User::getOwnerDataByNick` will work better.

Reply:
Hum, it isn't a technical endpoint. Still, most of the cruft of `Profile::load` is useless if it's intended to be parsed by a computer.

Reply:
Yes. My plan is also to provide a real barebone hcard-page in the future.